modules/openapi-generator/src/main/resources/typescript-fetch/modelGeneric.mustache

```diff
@@ -1,38 +1,20 @@
-/**
- * {{{description}}}
- * @export
- * @interface {{classname}}
- */
```

Comment:
I think this comment section should remain instead of the new one.

Reply:
sure thing